src/framebuffer.rs

```diff
@@ -41,32 +46,32 @@ pub const fn buffer_size_bpp(width: usize, height: usize, bpp: usize) -> usize {
 /// use embedded_graphics::{
 ///     framebuffer,
 ///     framebuffer::{Framebuffer, buffer_size},
-///     pixelcolor::{Rgb565, raw::LittleEndian},
+///     pixelcolor::{Rgb565, raw::storage::LittleEndian},
 ///     prelude::*,
 ///     primitives::PrimitiveStyle,
 /// };
 ///
-/// let mut fb = Framebuffer::<Rgb565, _, LittleEndian, 320, 240, {buffer_size::<Rgb565>(320, 240)}>::new();
+/// let mut fb = Framebuffer::<LittleEndian<Rgb565>, 320, 240, {buffer_size::<Rgb565>(320, 240)}>::new();
 ///
 /// fb.bounding_box()
 ///     .into_styled(PrimitiveStyle::with_stroke(Rgb565::RED, 1))
 ///     .draw(&mut fb)
 ///     .unwrap();
 /// ```
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
-pub struct Framebuffer<C, R, BO, const WIDTH: usize, const HEIGHT: usize, const N: usize> {
+pub struct Framebuffer<CS, DIR, const WIDTH: usize, const HEIGHT: usize, const N: usize> {
```

Comment:
`ImageRaw` uses a default `Horizontal` pixel arrangement, we could do the same for the `Framebuffer` if we move the `DIR` type parameter to the end.